configuration/concepts/registration_other_information.csv

```diff
@@ -0,0 +1,22 @@
+Uuid,Fully specified name:en,Fully specified name:fr,Short name:en,Short name:fr,Description:en,Description:fr,Data class,Data type,Answers
```

Comment:
@tmvumbi2 Is it necessary to add the uuids?

Reply:
Oh I see that is useful to add them because some of those concepts are answers to other concepts.